lib/pwwka/transmitter.rb

```diff
@@ -77,24 +79,40 @@ def self.send_message!(payload, routing_key,
             warn(resque_exception.message)
             raise e
           end
+
+        when :retry_async
```

Comment:
This is nearly identical to the `when` block above it. Is there a way to DRY this up? I don't think Ruby's case/when falls through.

Reply:
TIL you can use a comma for multiple options, so

```
when :resque, :retry_async
  # ...
```

will work if they are identical.

I guess another option is to leave the duplication and add a deprecation notice to the resque block (though that will probably destroy our logs with warnings…so maybe not :)